src/internal/scheduler/timerHandle.ts

```diff
@@ -0,0 +1 @@
+export type TimerHandle = number | NodeJS.Timeout;
```

Comment:
@sundbry This breaks compatibility with yarn v3 pnp: https://github.com/yarnpkg/berry/actions/runs/3521483267/jobs/5903377561